tests/core/pyspec/eth2spec/test/eip7594/unittests/polynomial_commitments/test_polynomial_commitments.py

```diff
@@ -48,12 +55,14 @@ def test_verify_cell_proof_batch(spec):
     commitment = spec.blob_to_kzg_commitment(blob)
     cells, proofs = spec.compute_cells_and_proofs(blob)
 
+    cells_bytes = [[field_element_bytes(element) for element in cell] for cell in cells]
+
     assert spec.verify_cell_proof_batch(
-        row_commitments=[commitment],
+        row_commitments_bytes=[commitment],
         row_ids=[0],
         column_ids=[0, 1],
-        cells=cells[0:1],
-        proofs=proofs,
+        cells_bytes=cells_bytes[0:1],
```

Comment:
How many cells are we trying to verify here? Since `column_ids` has two values, it appears to be two. But `[0:1]` will result in a single value. Think there's a bit of an accident here.

Reply:
Yeah that unittest was a bit working by accident. I made it more robust in 2000a4f30.

In the future we will want certainly want `test_verify_cell_proof_batch()` to do a multi-blob cell verification, but to do that we need to figure out a solution to `compute_cells_and_proofs()` taking multiple minutes (perhaps load them directly from a file?).